app-office/whid/whid-2.0.2~beta.recipe

```diff
@@ -0,0 +1,88 @@
+SUMMARY="Time Tracker for Freelancers and Individual Contractors"
+DESCRIPTION="What Have I Done - Time Tracker for Freelancers and Individual Contractors. \
+Features: \
+* Very easy and intuitive to use  \
+* Measures time for active tasks \
+* Maintains a tree-structures of folders, customers, projects and tasks \
+* Reports time spent, breakdowns for each day in a week \
+* Reports time spent on selected items in the tree structure of tasks \
+* Stores the data in SQLite \
+* Based on 20 years of experience and two previous iterations designing and implementing time \
+  tracking applications \
+* It's free, contains no ads, does not track you, and stores your data on your local computer. \
+"
+COPYRIGHT="1999-2019 Jarle Aase"
+HOMEPAGE="https://github.com/jgaa/whid"
+LICENSE="GNU GPL v3"
+REVISION="1"
+SOURCE_URI="https://github.com/jgaa/whid/archive/${portVersion/\~beta/-beta-01}.tar.gz"
+CHECKSUM_SHA256="b058ea8040ca6bf5dd2c6333b3966b77354a13e581cadd023dfbaf26ba3045a9"
+SOURCE_FILENAME="whid-${portVersion/\~beta/-beta-01}.tar.gz"
+SOURCE_DIR="whid-${portVersion/\~beta/-beta-01}"
+PATCHES="whid-${portVersion/\~beta/~beta}.patchset"
+ADDITIONAL_FILES="whid.rdef.in"
+ARCHITECTURES="!x86_gcc2 x86 x86_64"
+if [ "$targetArchitecture" = x86_gcc2 ]; then
+	SECONDARY_ARCHITECTURES="x86"
+fi
+
+PROVIDES="
+	whid$secondaryArchSuffix = $portVersion
+	app:Whid = $portVersion
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libGL$secondaryArchSuffix
+	lib:libQt5Core$secondaryArchSuffix
+	lib:libQt5Gui$secondaryArchSuffix
+	lib:libQt5Network$secondaryArchSuffix
```

Comment:
The generated executable doesn't depends on this.

Reply:
ok. remove that